docs/recipes/delete-files-folder.md

```diff
@@ -70,13 +70,20 @@ var del = require('del');
 var vinylPaths = require('vinyl-paths');
 
 gulp.task('clean:tmp', function () {
+  var vp = vinylPaths();
   return gulp.src('tmp/*')
+    .pipe(vp)
```

Comment:
Why not just move `.pipe(vinylPaths(del));` here? It's read into the disk anyways. So we can just delete it right away.